test/Main.purs

```diff
@@ -61,7 +60,7 @@ spec = do
       testComponent def do
         find ".baz" >> do
           prop "value" >>= shouldEqual ""
-          simulate' "change" $ unsafeToForeign { target: { value: "New text" } }
+          simulate' "change" { target: { value: "New text" } }
```

Comment:
According to [Enzyme docs](https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/simulate.html), this parameter is not just any old value that gets passed straight to the handler. It's a record that gets merged into the synthetic event. So I changed it from `a` to `Record r`.

And this is also (I'm guessing) the reason that `simulate` only works for "standard" elements, not for custom events.

Reply:
Oh didn’t realize, good catch